integration_tests/src/main/scala/com/nvidia/spark/rapids/tests/common/BenchUtils.scala

```diff
@@ -584,6 +629,15 @@ case class SparkSQLMetric(
     metricType: String,
     value: Any)
 
+/** Summary of stage-level metrics */
+case class StageMetrics(
```

Comment:
nit - its a little bit confusing with the name of this because it looks like a Spark name.  like StageInfo. I wonder if we should add something to the name just to be easier to differentiate. Its not that big of a deal though.